### ISSUE TITLE
Mod errors

### ISSUE THREAD

**Issue by Alucard0Reborn** [DW8030/Civ6-mod#20]
Catapult obsoletes at tech Military Engineering, and therefore Metal Casting for its upgrade Bombard may not yet be researched!
Domrey obsoletes at tech Metal Casting, and therefore Steel for its upgrade Artillery may not yet be researched!
Frigate obsoletes at tech Steel, and therefore Refining for its upgrade Battleship may not yet be researched!
De Zeven Provincien obsoletes at tech Steel, and therefore Refining for its upgrade Battleship may not yet be researched!
Jong obsoletes at tech Steel, and therefore Refining for its upgrade Battleship may not yet be researched!

**Comment by DW8030** [comment]
You are right. I thought I had made sure all the upgrade paths were correct. I made an update that should fix the problem. Thank you.
